github-pullrequest-plugin/src/main/resources/org/jenkinsci/plugins/github/pullrequest/GitHubPRTrigger/config.groovy

```diff
@@ -39,8 +39,10 @@ f.block {
             )
         }
 
-        f.optionalProperty(title: "User Restriction", field: "userRestriction")
-   
-        f.optionalProperty(title: "Branch Restriction", field: "branchRestriction")
+        f.advanced() {
```

Comment:
I just think it's best to hide them out of the way ASAP given that we don't know if they actually do anything. =) But your call! =)